lemur/plugins/lemur_email/templates/rotation.html

```diff
@@ -98,7 +98,7 @@
                                             </tr>
```

Comment:
I "fixed" this file a while ago, meaning I got it to successfully render. Looking at it again today, the contents were not correct. This new version should be closer to what we actually want out of it.

Reply:
<img width="532" alt="Screen Shot 2021-05-17 at 6 47 20 PM" src="https://user-images.githubusercontent.com/453663/118578041-56fad080-b740-11eb-98dc-0dad492c0c32.png">
<img width="547" alt="Screen Shot 2021-05-17 at 6 47 14 PM" src="https://user-images.githubusercontent.com/453663/118578044-582bfd80-b740-11eb-9cfa-6ddb520b96c7.png">